tests/testing_common.py

```diff
@@ -284,7 +285,7 @@ def _test_merge_layers(self, model_id, config_cls, config_kwargs):
         if config.peft_type != "LORA":
             with self.assertRaises(AttributeError):
                 model = model.merge_and_unload()
-        elif model.config.model_type == "gpt2":
+        elif model.config["model_type"] == "gpt2":
```

Comment:
should be resolved by the Conv1D support you have added, so no need for this.

Reply:
Yes, this works now for LoRA, unfortunately not for IA³. Not sure why, but it didn't look like a quick fix to me.